README.md

```diff
@@ -1,2 +1,38 @@
 # Sf2-OoT-parser
 Series of scripts that process a soundfont 2 file and a sequence file into the Ocarina of Time Randomizer music format
+
```

Comment:
i do really like the way the readme file here looks, i don't think i have any issues with it